### PR TITLE
Upgrade redmine from 3.4.7 to 4.0.0

### PR DESCRIPTION
**Pull request by obsessionsys**
Hello!
This is my first  pull request. 
Updated redmine to version 4.0.0

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,6 +44,8 @@ RUN apt-get update \
  && gem install --no-document bundler \
  && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
```
Comment (jcormier) [inline Dockerfile]: Can you move this change to a different commit since its not related to the 4.0 migration?  Also, add a comment describing what its doing.

**Comment by jcormier** [comment]
Thanks for this pull request.

**Comment by jcormier** [comment]
As far as I can tell adding the purge hasn't changed the list of installed packages
```diff
 $ diff -u dpkg_3.4.7-1.txt dpkg_4.0.0.txt
--- dpkg_3.4.7-1.txt	2019-01-16 09:41:07.761452506 -0500
+++ dpkg_4.0.0.txt	2019-01-16 09:41:25.625107723 -0500
@@ -248,7 +248,7 @@
 ii  libxrender1:am 1:0.9.9-0ubu amd64        X Rendering Extension client libr
 ii  libxslt1.1:amd 1.1.28-2.1ub amd64        XSLT 1.0 processing library - run
 ii  libyaml-0-2:am 0.1.6-3      amd64        Fast YAML 1.1 parser and emitter 
-ii  linux-libc-dev 4.4.0-140.16 amd64        Linux Kernel Headers for developm
+ii  linux-libc-dev 4.4.0-141.16 amd64        Linux Kernel Headers for developm
 ii  locales        2.23-0ubuntu all          GNU C Library: National Language 
 ii  login          1:4.2-3.1ubu amd64        system login tools
 ii  logrotate      3.8.7-2ubunt amd64        Log rotation utility
@@ -309,7 +309,7 @@
 ii  sysv-rc        2.88dsf-59.3 all          System-V-like runlevel change mec
 ii  sysvinit-utils 2.88dsf-59.3 amd64        System-V-like utilities
 ii  tar            1.28-2.1ubun amd64        GNU version of the tar archiving 
-ii  tzdata         2018g-0ubunt all          time zone and daylight-saving tim
+ii  tzdata         2018i-0ubunt all          time zone and daylight-saving tim
 ii  ubuntu-keyring 2012.05.19   all          GnuPG keys of the Ubuntu archive
 ii  ucf            3.0036       all          Update Configuration File(s): pre
 ii  util-linux     2.27.1-6ubun amd64        miscellaneous system utilities
```

**Comment by jcormier** [comment]
Built container and ran for postgres and mysql
[docker-compose_up.txt](https://github.com/sameersbn/docker-redmine/files/2765064/docker-compose_up.txt)

**Comment by obsessionsys** [comment]
I apologize for close this request, trying to fix it
